.ci/jenkins/tests/pom.xml

```
@@ -46,7 +46,7 @@
             ${JENKINS_URL}/systemInfo
             Please do not use different from what is in Jenkins unless really necessary
         -->
-        <version.jenkins>2.426.3</version.jenkins>
+        <version.jenkins>2.452.4</version.jenkins>
```

we don't use 2.452.4 version, but 2.440.2
```suggestion
        <version.jenkins>2.440.2</version.jenkins>
```